src/test.rs

```diff
@@ -131,11 +131,6 @@ fn get_containers_response() -> String {
 }
 
 #[cfg(test)]
-fn get_containers_response_long() -> String {
```

Comment:
Ah, I see. It's a test support function but not used anywhere. OK, it's safe to delete.